client/asset/dcr/dcr.go

```diff
@@ -5386,6 +5386,21 @@ func (dcr *ExchangeWallet) monitorPeers(ctx context.Context) {
 	}
 }
 
+func (dcr *ExchangeWallet) emitTipChange(height int64) {
+	var data interface{}
+	// stakeInfo, err := dcr.wallet.StakeInfo(dcr.ctx)
+	// if err != nil {
+	// 	dcr.log.Errorf("Error getting stake info for tip change notification data: %v", err)
+	// } else {
+	// 	data = &struct {
+	// 		TicketPrice uint64 `json:"ticketPrice"`
+	// 	}{
+	// 		TicketPrice: uint64(stakeInfo.Sdiff),
+	// 	}
+	// }
+	dcr.emit.TipChange(uint64(height), data)
```

Comment:
I leave this here as a demonstration of how wallets can use these to convey asset-customized info when needed. Another use case is for asynchronous ticket purchasing, which may be necessary when mixing is enabled (or even for unmixed tickets, possibly). The `StakeInfo` method isn't added in this PR, but it is in the gui staking PR #2482.